addons/mail/models/mail_render_mixin.py

```diff
@@ -284,10 +285,10 @@ def _render_template_qweb(self, template_src, model, res_ids,
                 )
                 # remove the rendered tag <div> that was added in order to wrap potentially multiples nodes into one.
                 render_result = render_result[5:-6]
-            except QWebCodeFound:
-                group = self.env.ref('mail.group_mail_template_editor')
-                raise AccessError(_('Only users belonging to the "%s" group can modify dynamic templates.', group.name))
             except Exception as e:
+                if isinstance(e, QWebException) and isinstance(e.error, QWebCodeFound):
```

Comment:
I don't know if there is anything to do about this, but it seems to me that having 2 nesting exceptions (`e` is a exception and `e.error` is an exception) is weird.

Reply:
Mail triggers the error and as it occurs in the qweb it is nested. The qweb is the errors to add information such as the template and the problematic tag

Reply:
I agree that exceptions in exceptions is weird... and it does have some far implications, see https://github.com/odoo/odoo/blob/cc8ebed3c66b0a97517da76155b32b3174e9e412/addons/http_routing/models/ir_http.py#L581-L585